src/routes/v0/endpoints.rs

```diff
@@ -54,10 +54,10 @@ pub const SEARCH_TAGS_ROUTE: &str = concatcp!(SEARCH_PREFIX, "/tags/{label}");
 
 // TAG endpoints
 // Axum routes
-const TAG_PREFIX: &str = concatcp!(VERSION_ROUTE, "/tag");
-pub const TAG_HOT_ROUTE: &str = concatcp!(TAG_PREFIX, "/hot");
-pub const TAG_REACH_ROUTE: &str = concatcp!(TAG_PREFIX, "/reached/{user_id}/{reach}");
-pub const TAG_TAGGERS_ROUTE: &str = concatcp!(TAG_PREFIX, "/taggers/{label}");
+const TAG_PREFIX: &str = concatcp!(VERSION_ROUTE, "/tags");
+pub const HOT_TAGS_ROUTE: &str = concatcp!(TAG_PREFIX, "/hot");
+pub const HOT_TAGS_BY_REACH_ROUTE: &str = concatcp!(HOT_TAGS_ROUTE, "/{user_id}/{reach}");
```

Comment:
1. Only `HOT_TAGS_ROUTE` is needed, the reach and timeframe options should all be available in the same endpoint.
2. Therefore `user_id` and `reach` should become a query params.
3. Feels like starting with `HOT` instead of `TAG` breaks our endpoint `const` naming convention.